com.unity.render-pipelines.universal/Editor/Camera/UniversalRenderPipelineCameraEditor.cs

```diff
@@ -251,44 +251,63 @@ void DrawElementCallback(Rect rect, int index, bool isActive, bool isFocused)
-                    if (!errorCameras.Contains(cam))
+                    typeError = true;
+                    if (!m_TypeErrorCameras.Contains(cam))
                     {
-                        errorCameras.Add(cam);
+                        m_TypeErrorCameras.Add(cam);
                     }
                 }
-                else if (errorCameras.Contains(cam))
+                else if (m_TypeErrorCameras.Contains(cam))
                 {
-                    errorCameras.Remove(cam);
+                    m_TypeErrorCameras.Remove(cam);
                 }
 
-                var labelWidth = EditorGUIUtility.labelWidth;
-                EditorGUIUtility.labelWidth -= 20f;
-                if (warning)
+                bool outputWarning = false;
+                if (IsStackCameraOutputDirty(cam))
                 {
-                    GUIStyle errorStyle = new GUIStyle(EditorStyles.label) { padding = new RectOffset { left = -16 } };
-                    m_NameContent.text = cam.name;
-                    EditorGUI.LabelField(rect, m_NameContent, TempContent(type.GetName(), warningInfo, m_ErrorIcon), errorStyle);
+                    outputWarning = true;
+                    if (!m_OutputWarningCameras.Contains(cam))
+                    {
+                        m_OutputWarningCameras.Add(cam);
+                    }
                 }
-                else
+                else if (m_OutputWarningCameras.Contains(cam))
                 {
-                    EditorGUI.LabelField(rect, cam.name, type.ToString());
+                    m_OutputWarningCameras.Remove(cam);
+                }
 
-                    // Printing if Post Processing is on or not.
-                    var isPostActive = cam.gameObject.GetComponent<UniversalAdditionalCameraData>().renderPostProcessing;
-                    if (isPostActive)
-                    {
-                        Rect selectRect = new Rect(rect.width - 20, rect.y, 50, EditorGUIUtility.singleLineHeight);
+                GUIContent nameContent =
+                    outputWarning ?
+                    EditorGUIUtility.TrTextContent(cam.name, "Output properties do not match base camera", m_WarningIcon) :
+                    EditorGUIUtility.TrTextContent(cam.name);
 
-                        EditorGUI.LabelField(selectRect, "PP");
-                    }
+                GUIContent typeContent =
+                    typeError ?
+                    EditorGUIUtility.TrTextContent(type.GetName(), "Not a supported type", m_ErrorIcon) :
+                    EditorGUIUtility.TrTextContent(type.GetName());
+
+                EditorGUI.BeginProperty(rect, GUIContent.none, element);
+                var labelWidth = EditorGUIUtility.labelWidth;
+                EditorGUIUtility.labelWidth -= 20f;
+
+                Vector2 cachedIconSize = EditorGUIUtility.GetIconSize();
+                EditorGUIUtility.SetIconSize(new Vector2(rect.height, rect.height)); // To prevent icon shrinking
```

Comment:
minor:You have the scope `new EditorGUIUtility.IconSizeScope`

Reply:
noice, fixed